todoman/cli.py

```diff
@@ -290,6 +302,7 @@ def move(ctx, list, ids):
 @click.option('--location', help='Only show tasks with location containg TEXT')
 @click.option('--category', help='Only show tasks with category containg TEXT')
 @click.option('--grep', help='Only show tasks with message containg TEXT')
+# XXX: shouldn't this be an ngarg-1 this?:
```

Comment:
I mean: shouldn't `--sort` be a [multi value option](http://click.pocoo.org/6/options/#multi-value-options)? We actually split the string manually ourselves below.